datafusion/core/src/physical_plan/file_format/mod.rs

```diff
@@ -796,4 +808,38 @@ mod tests {
             output_ordering: None,
         }
     }
+
+    #[test]
+    fn file_groups_display_empty() {
+        let expected = "";
+        assert_eq!(&FileGroupsDisplay(&vec![]).to_string(), expected);
+    }
+
+    #[test]
+    fn file_groups_display() {
```

Comment:
Prior to the changes in this PR, this was displayed as `"[foo, bar, baz, ]"`


I think this PR is much clearer